### PR TITLE
update chart data path

### PR DESCRIPTION
**Pull request by vasba**

### CODE REVIEW

**Comment by Qluxzz** [comment]
Do the arguments (period and resolution) still work as expected, and does the return model look the same as before?

Otherwise, these need to be updated as well.

**Comment by vasba** [comment]
The period and resolution still work but only certain resolutions are available for a given period.

I have updated the returned model.

**Review by AmirEghbali — APPROVED** [review]

**Review by AmirEghbali — COMMENTED** [review]
I cannot get this work. Can you please copy and paste a code example?

**Comment by vasba** [comment]
@AmirEghbali 
This commit https://github.com/Qluxzz/avanza/commit/ee99bf7048bf7b9353563b1b5f5b5342a14cf23d done by you removed some periods that are needed to this endpoint.

I will have to add back the periods that you removed because I use them.

**Comment by AmirEghbali** [comment]
Please add them back. When I removed some periods it was because they were
not supported  in insights_report.

Den tors 22 dec. 2022 20:34vasba ***@***.***> skrev:

**Comment by AmirEghbali** [comment]
This should close #68. Right?

Den tors 22 dec. 2022 20:34vasba ***@***.***> skrev:

**Comment by vasba** [comment]
@AmirEghbali 

Yes it does!

Who can merge this?

File: avanza/constants.py [hunk 1]
```diff
@@ -91,7 +96,7 @@ class Route(enum.Enum):
     ACCOUNT_OVERVIEW_PATH = '/_mobile/account/{}/overview'
     ACCOUNTS_POSITIONS_PATH = '/_api/position-data/positions'
     AUTHENTICATION_PATH = '/_api/authentication/sessions/usercredentials'
-    CHARTDATA_PATH = '/_mobile/chart/orderbook/{}'
+    CHARTDATA_PATH = '/_api/price-chart/stock/{}'
```
Comment (Qluxzz) [inline avanza/constants.py:99]: The `get_chart_data` which uses this api url takes an order book id which can be anything from a stock, a fund, a warrant.

Does the new url work with all types of instruments or just stocks?

Comment (vasba) [reply avanza/constants.py:99]: I have just tested it works for stock, warrants, index and funds.

**Comment by Qluxzz** [comment]
Thanks for contributing!

**Pull request merged**